docs/analytics/apis/lookup-tables-api.md

```diff
@@ -0,0 +1,236 @@
+---
+title: Lookup Table API
+description: The Lookup Table API lets you create, get, update, and delete lookup tables to augment your data.
+---
+
+!!!beta "This feature is in beta"
+    This requires that you have the `lookup_table` feature enabled in Amplitude. Ask your rep to help you with this.
+
+### Introduction
+
+Lookup Tables is a way to augment properties. Instead of using formulas, you can upload a CSV file that contains property mappings to derive new properties.
+
+To create a Lookup property, you need to create a Lookup Table to reference. You can retrieve and update each of the tables using the API. Lookup Tables are identified by the name and are scoped per project.
+
```

Comment:
Please add an endpoint section, like this one: https://www.docs.developers.amplitude.com/analytics/apis/behavioral-cohorts-api/#endpoints

```
## Endpoints

| Region | Endpoint |
| --- | --- |
| Standard Server | <ENDPOINT HERE> |
| EU Residency Server | <EU ENDPOINT HERE DELETE ROW IF NOT NEEDED> |
```